sample_app/src/main/java/com/discord/sampleapp/MainActivity.kt

```diff
@@ -70,9 +70,7 @@ class MainActivity : AppCompatActivity(),
     // This will also work in other cases like child views in Fragments within MainActivity
     // because PanelsChildGestureRegionObserver.Provider.get() returns an Activity-scoped
     // singleton.
-    horizontalScrollItemsContainer.addOnLayoutChangeListener(
```

Comment:
This old API wasn't working fully, so I updated this to use the new `register` API.